src/less/input-chips/input-chips.less

```diff
@@ -0,0 +1,119 @@
+@import "../variables/variables.less";
+
+.input-chips__container {
+    background-color: var(--color-background-secondary);
+    border: 1px solid var(--color-stroke-default);
+    border-radius: var(--border-radius-50);
+    display: flex;
+    flex-direction: column;
+    padding: var(--spacing-100) var(--spacing-200) var(--spacing-100)
+        var(--spacing-100);
+    position: relative;
+}
+
+.input-chips__content {
+    display: flex;
+    flex-direction: row;
+    gap: var(--spacing-100);
+    margin-bottom: var(--spacing-100);
+}
+
+.input-chips_items {
+    display: flex;
+    flex: auto;
+    flex-wrap: wrap;
+    gap: var(--spacing-100);
+    max-width: 100%;
+}
+
+.input-chips label {
+    display: inline-block;
+    font-size: var(--font-size-default);
+    margin-bottom: var(--spacing-50);
+}
+
+.input-chips__helper-text {
+    color: var(--color-foreground-secondary);
+    font-size: var(--font-size-small);
+    margin-top: var(--spacing-100);
+}
+
+.input-chips button.icon-btn {
+    background-color: transparent;
+}
+
+.input-chips input {
+    background-color: inherit;
+    border: none;
+    border-radius: 0;
+    height: auto;
+    margin: 0 8px;
+    padding: 6px 0;
+    width: calc(100% - 16px);
+}
+
+.input-chips .combobox {
+    margin-left: -8px;
+    margin-top: 8px;
+    width: calc(100% + 24px);
+}
+
+.input-chips .combobox__listbox {
+    background-color: var(--color-background-elevated);
+    border: none;
+    border-radius: 16px;
+    box-shadow: var(--shadow-strong);
```

Comment:
Should we really be deviating this much from core listbox styles? I would imagine that the only part we should override is the positioning.

Reply:
If DS has the current specs where they have it, and we're trying to make it look how they want, which is a purely visual "preference," should the fact that we've chosen to use combobox under the hood be enough to overturn the visual specs that DS has landed on?